apps/designer/app/shared/sync/sync-stores.ts

```diff
@@ -69,8 +80,74 @@ const syncChanges = (name: SyncEventSource, publish: Publish) => {
   };
 };
 
+const syncStoresState = (name: SyncEventSource, publish: Publish) => {
+  const latestData = new Map<string, unknown>();
+
+  const unsubscribeRemoteChanges = subscribe(
+    "sendStoreData",
+    ({ source, data }) => {
+      /// prevent reapplying own changes
+      if (source === name) {
+        return;
+      }
+      for (const { namespace, value } of data) {
+        // apply immerhin stores data
+        const container = store.containers.get(namespace);
+        if (container) {
+          container.set(value);
+        }
+        // apply state stores data
+        const stateStore = stateStores.get(namespace);
```

Comment:
thinking about a better name than stateStore, what about uiStore and uiStores?

Reply:
clientOnlyStore?